### DISCUSSION TITLE
How to get the svg path of the graphic

### DISCUSSION THREAD

**Discussion by KaboChow** [pymupdf/PyMuPDF#3695] [Q&A] [answered]
Hello, I would like to use PyMuPDF to recognize the shapes in a PDF and render them onto a Fabric.js canvas. The Fabric.js path object supports a path format similar to the SVG <path> element, for example:
`var canvas = new fabric.Canvas('c');
var path = new fabric.Path('M 0 0 L 200 100 L 170 200 z');
path.set({ left: 120, top: 120 });
canvas.add(path);`
The paths obtained using get_drawings are not suitable. How can I get paths in this format: 'M 0 0 L 200 100 L 170 200 z'?

**Comment by JorjMcKie** [comment, marked as answer]
There also is `page.get_svg_image()` which delivers a string in SVG format.
This might help you.
